framework/crosscutting/events/storeKafka.ts

```diff
@@ -0,0 +1,112 @@
+import { Kafka } from 'kafkajs';
+
+import Metadata from './integration/metadata';
+import EventAttiv from './integration/eventAttiv';
+import Attivlogger from '../logging/logger';
+import messages from '../messages/message';
+import IStoreBase from './integration/IStoreBase';
+
+export default class StoreSQS implements IStoreBase {
+  private subscribes: Array<EventAttiv> = [];
+
+  private consumers: Object = {};
+
+  private kafka: any;
```

Comment:
I think this should be typed as `Kafka` 🤔

Reply:
Makes sense, I changed it.